packages/frontend/src/constants.ts

```diff
@@ -23,22 +23,37 @@ export interface CarrotChain extends SupportedChain {
     icon: ChainIconData;
 }
 
-export const SUPPORTED_CHAINS: [CarrotChain, ...CarrotChain[]] = [
-    {
-        ...SUPPORTED_CHAIN[ChainId.Sepolia],
-        icon: {
-            logo: EthereumLogo,
-            backgroundColor: "#8637ea",
-        },
-    },
-    {
-        ...SUPPORTED_CHAIN[ChainId.ArbitrumSepolia],
-        icon: {
-            logo: ArbitrumLogo,
-            backgroundColor: "#213147",
-        },
-    },
-] as const;
+export const SUPPORTED_CHAINS: [CarrotChain, ...CarrotChain[]] =
+    __ENVIRONMENT__ === "dev"
```

Comment:
Is this `__ENVIRONMENT__` initialized globally somewhere?

Reply:
It's passed as a global in the craco config. There's also a check that enforces its valid definition.